src/txn/txn_recover.c

```diff
@@ -1016,6 +1016,9 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         WT_ERR(__wt_rollback_to_stable(session, NULL, true));
     }
 
+    if (eviction_started)
```

Comment:
Can you add a comment here to explain the reason behind stopping the eviction workers before doing the final checkpoint at end of recovery? Otherwise, we may forget later.

Reply:
Thanks for the suggestion Hari.